packages/apps/graph/src/devnet/deployment/marmalade/deploy.ts

```diff
@@ -78,9 +81,14 @@ export async function deployMarmaladeContracts(
     file.endsWith(marmaladeRemoteConfig.templateExtension),
   );
 
+  const upgradeTemplateFiles = templateFiles.filter((f) => {
+    return f.split('.')[1] === '8' || f.split('.')[3] === 'guard-policy-v1';
+  });
+
```

Comment:
Just to be sure, the files that match this criteria are to be deployed first and then upgraded?

Reply:
Yes, they're to be deployed with `upgrade=false` then deployed with `upgrade=true`